base/src/expressions/parser/mod.rs

```diff
@@ -111,7 +112,10 @@ pub enum Node {
     },
     RangeKind {
         sheet_name: Option<String>,
+        // index into vector (0-based)
```

Comment:
Yes, they are, but I feel we should document this at the API level and not clutter code here.

Kind of the leitmotif is that a cell is referenced by three numbers `(sheet, row, column)` in that order when sheet is an index (0-indexed) and (row, column) are 1 based. But this is not the place to document this. Methinks